Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json

```diff
@@ -0,0 +1,408 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Digital Shadows Playbook to Update Incident Status",
+        "description":  "The playbook will update the status of Azure incidents which are created from the Alerts triggered by Digital Shadows SearchLight portal",
+        "prerequisites":  "The Digital Shadows Data Connector is deplyed on your environment",
```

Comment:
Could we add a 'mainSteps' section here that describes what the workbook does?

Example: https://github.com/Azure/Azure-Sentinel/blob/master/Playbooks/CiscoASA/CiscoASA-AddIPtoNetworkObjectGroup/azuredeploy.json

```
"mainSteps": [
  "1. This playbook triggers when a Microsoft Sentinel Incident is created or updated by the Digital Shadows Data Connector",
  "2. The status of the inbound Digital Shadows alert is inspected and the Microsoft Sentinel Incident status is updated to match"
]
```

I think this is what it does - if I've got that wrong or missed something out, please comment